pontoon/base/views.py

```diff
@@ -1,8 +1,8 @@
 import logging
 import os
+from datetime import datetime
 
 from bulk_update.helper import bulk_update
-from datetime import datetime
```

Comment:
I'm curious, why did you move this?

Reply:
Well ``datetime`` is part of the standard library, so it should be with the other imports from that standard library, at the top.

Reply:
Oh, OK. For some reason I got misled by the `from` into thinking this is not a standard library, but `form` has nothing to do with it of course.